### PR TITLE
Ec course assistants

### PR DESCRIPTION
**Pull request by ericthewry**
Heres the guide for course assistants.  Let me know if you have any feedback @rwoll

### CODE REVIEW

File: course_assistants.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+record their grade in the Google Doc. *Do not delete the branch.*
+
+If a student has questions or concerns about their grade, they will be encouraged
+to leave comments and questions about your comments. If you are unsure about how
+to grade someone's code, you can always tag another TA or Professor in an inline
+comment.
+
+> Note: It is currently undecided if/how Git will be incorporated into the
+> students' grades. If you have suggestions, please [join the discussion](link to
+> issue).
```
Comment (rwoll) [inline course_assistants.md]: This doc looks good. Should we make it clearer that the cloning only occurs once per semester for each student (at least ideally.) As part of the grading workflow, TAs will see the repos in the EGit window. After they have `pull`ed in the new branches and commits from students, they will need to right click and import in the EGit window on the the student repo to switch to grading that person's code.

@ericthewry

**Comment by rwoll** [comment]
I'm going to close this, but leave the branch alive so we can move the guides.

**Comment by ericthewry** [comment]
@rwoll let me know if this works. and holy crap windows....

**Comment by rwoll** [comment]
looks good! :+1:

**Pull request merged**